core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-tables/contributor-admin-stats-table.component.ts

```diff
@@ -121,6 +120,239 @@ export class ContributorAdminStatsTable implements OnInit {
     }
   }
 
+  pluralise(num: number, word: string): string {
+    return num === 1 ? word : word + 's';
+  }
+
+  getCountDisplay(
+      count: number,
+      wordCount: number = 0,
+      editDisplay: string = ''): string {
+    return count + this.pluralise(count, ' card') +
+           editDisplay +
+         (
+          wordCount === 0 ?
+                       '' :
+                       ', ' + wordCount + this.pluralise(wordCount, ' word')
+         );
+  }
+
+  getAcceptedSubmittedDisplay(
+      acceptedTranslationCount: number,
+      acceptedTranslationsWithoutReviewerEditsCount: number,
+      acceptedTranslationWordCount: number = 0): string {
+    return this.getCountDisplay(
+      acceptedTranslationCount,
+      acceptedTranslationWordCount,
+      ' (' + acceptedTranslationsWithoutReviewerEditsCount +
+        ' without edits)');
+  }
+
+  getAcceptedReviewedDisplay(
+      acceptedTranslationCount: number,
+      acceptedTranslationsWithReviewerEditsCount: number,
+      acceptedTranslationWordCount: number = 0): string {
+    return this.getCountDisplay(
+      acceptedTranslationCount,
+      acceptedTranslationWordCount,
+      ' (' + acceptedTranslationsWithReviewerEditsCount + ' edited)');
+  }
+
+  getActiveTopicDisplay(topics: string[]): string {
```

Comment:
Perhaps this could be described more clearly. I think the issue is that "display" isn't really the same as a string, it is reminiscent of a TV display or something like that. Maybe "getTextForXXHeader" or whatever the relevant UI element is.

Reply:
Done. Display is used in updateColumnsForDisplay, I didn't think it through enough bout how these methods are used differently. I've changed it to textForAttribute throughout the new methods.